core/templates/domain/question/question-backend-api.service.ts

```diff
@@ -113,17 +106,15 @@ export class QuestionBackendApiService {
   }
 
   private _fetchQuestionSummaries(
-      skillIds: string[], cursor: string,
+      skillId: string, cursor: string,
       successCallback: (value: QuestionSummariesResponse) => void,
       errorCallback: (reason: string) => void): void|boolean {
-    if (!this.isListOfStrings(skillIds)) {
```

Comment:
Why remove this?

Reply:
The argument was changed to be a string instead of an array.